C/mcal/GPIO/GPIO.c

```diff
@@ -0,0 +1,73 @@
+#include "../../common/Types.h"
+#include "../../common/Config.h"
+#include "../../common/Registes.h"
+#include "GPIOPrivate.h"
+#include "GPIO.h"
+
+
+Error_t GPIO_SetPinDirection(PORT_t PortNumber , PIN_t PinNumber , DIRECTION_t Direction)
+{
+	Error_t LocReturnError = NoError ;
```

Comment:
I really hate this style (IMT style) all developers also in `g2` know this is a local variable. I think you won't know this if the reader is a barber. 😆 
`error_t returnError`